ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-ruby31-v1.14.x.yaml

```diff
@@ -86,7 +86,7 @@ base_images:
   ubi8-ruby31:
```

Comment:
Should this be `ubi9-ruby31`? Similarly below where it is used.

Reply:
its just a name but i will change it for correctness